assets/wizards/onboarding/views/locationSetup.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Location setup Screen.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Card, FormattedHeader, Button, TextControl, SelectControl } from '../../../components/src';
+
+/**
+ * Location Setup Screen.
+ */
+class LocationSetup extends Component {
+	/**
+	 * Handle an update to a setting field.
+	 *
+	 * @param string key Setting field
+	 * @param mixed  value New value for field
+	 *
+	 */
+	handleOnChange( key, value ) {
+		const { location, onChange } = this.props;
+		location[ key ] = value;
+		onChange( location );
+	}
+
+	/**
+	 * Render.
+	 */
+	render() {
+		const { location, onClickContinue, onClickSkip } = this.props;
+		const { countrystate, address1, address2, city, postcode, currency } = location;
+
+		return (
+			<div className="newspack-location-setup-screen">
+				<FormattedHeader
+					headerText={ __( 'About your publication' ) }
+					subHeaderText={ __(
+						'This information is required for accepting payments and other features'
+					) }
+				/>
+				<Card>
+					<SelectControl
+						label={ __( 'Where is your business based?' ) }
+						value={ countrystate }
+						options={ newspack_location_info }
```

Comment:
Do we want to autocomplete the location and currency settings or something? Currently they are giant selects with 100+ options.

Reply:
Not needed. See comment below.